src/components/TableView.ts

```diff
@@ -0,0 +1,61 @@
+import * as vscode from "vscode";
+
+export interface Row {
+  Field: Array<string>;
+}
+
+export class TableView {
+  public static show(rows: Array<Row>, title: string) {
+    const panel = vscode.window.createWebviewPanel(
+      "InfluxDB",
+      title,
+      vscode.ViewColumn.Two,
+      {
+        retainContextWhenHidden: true
+      }
+    );
+
+    panel.webview.html = TableView.getWebviewContent(rows);
+  }
+
+  public static getWebviewContent(rows: Array<Row>): string {
+    const head = `<!DOCTYPE html>
```

Comment:
could we move this markup and use `fs.readFileSync` at the top of the file to load it in as a variable?